hack/build/ci/get-image-digest.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
```

Comment:
IMO we should store the variables in env variables in this script and then return them from the step using the `outputs` field to make it more explicit where these values come from.

Reply:
I agree that the get-image-digest is not pretty and should be a seperate action, however i would tackle that in a seperate PR